proxy/server/manager.go

```diff
@@ -40,7 +40,7 @@ func LoadAndCreateManager(cfg *models.Proxy) (*Manager, error) {
 	if cfg.ConfigType == models.ConfigFile {
 		root = cfg.FileConfigPath
 	}
-	namespaceConfigs, err := loadAllNamespace(cfg.ConfigType, cfg.CoordinatorAddr, cfg.UserName, cfg.Password, root)
+	namespaceConfigs, err := loadAllNamespace(cfg.ConfigType, cfg.CoordinatorAddr, cfg.UserName, cfg.Password, root, cfg.EncryptKey)
```

Comment:
how about cfg as the param of loadAllNamespace @eastfisher #18

Reply:
I think loadAllNamespace() needs refactoring

Reply:
I change params to cfg,we can refactor some code in another issue.